app/pages/sections/order-items/associated-services/order-item/flat/ondemand/manifest.json

```diff
@@ -0,0 +1,79 @@
+{
+    "questions": {
+      "quantity": {
+        "id": "quantity",
+        "mainAdvice": "How many do you think you'll need?",
+        "rows": 3,
+        "expandableSection": {
+          "dataTestId": "view-section-quantity-id",
+          "title": "What quantity should I enter?",
+          "innerComponent": "This figure should relate to the entire duration of the order and not be an annual or monthly estimate. For example, if you want to order training to cover a three year period, with 10 training days per year, you would enter 30 days as the quantity."
+        }
+      },
+      "selectEstimationPeriod": {
+        "id": "selectEstimationPeriod",
+        "type": "radio",
+        "mainAdvice": "Over what period did you estimate the quantity?",
+        "options": [
+          {
+            "value": "month",
+            "text": "Per month"
+          },
+          {
+            "value": "year",
+            "text": "Per year"
+          }
+        ],
+        "expandableSection": {
+          "dataTestId": "view-section-estimation-period-id",
+          "title": "What period should I enter?",
+          "innerComponent": "This should be based on how you estimated the quantity you want to order."
+        }
+      }
+    },
+    "addPriceTable": {
+      "columnInfo": [
+        {
+          "data": "What price have you agreed? (£)",
+          "width": "50%",
+          "classes": "nhsuk-u-font-size-16"
+        },
+        {
+          "data": "Unit of order",
+          "width": "50%",
+          "classes": "nhsuk-u-font-size-16"
+        }
+      ],
+      "cellInfo": {
+        "price": {
+          "question": {
+            "type": "input",
+            "id": "price"
+          },
+          "classes": "nhsuk-input--width-10",
+          "expandableSection": {
+            "dataTestId": "view-section-input-id",
+            "title": "What price should I enter?",
+            "innerComponent": "You can change the list price if you’ve agreed a different rate with the supplier."
+          }
+        },
+        "unitOfOrder": {
+          "dataTestId": "unit-of-order"
+        }
+      }
+    },
+    "errorMessages": {
+      "QuantityRequired": "Enter an estimated quantity",
+      "QuantityMustBeANumber": "Quantity must be a number",
+      "QuantityInvalid": "Estimated quantity must be a whole number",
+      "QuantityGreaterThanZero": "Estimated quantity must be greater than zero",
+      "QuantityLessThanMax": "Maximum quantity exceeded, enter a lower estimated quantity",
+      "EstimationPeriodRequired": "Select an estimation period",
+      "PriceRequired": "Enter a price",
+      "PriceMustBeANumber": "Price must be a number",
+      "PriceMoreThan3dp": "Price must be to a maximum of 3 decimal places",
+      "PriceGreaterThanOrEqualToZero": "Price cannot be negative",
+      "PriceLessThanMax": "Maximum price exceeded, enter a lower price"
+    }
+  }
+  
```

Comment:
Not sure if it's the indenting but the braces don't line up and also git is complaining about no new line at end of file